api/src/pcapi/local_providers/cinema_providers/cds/cds_stocks.py

```diff
@@ -134,3 +207,19 @@ def get_next_product_id_from_database():  # type: ignore [no-untyped-def]
 def get_next_offer_id_from_database():  # type: ignore [no-untyped-def]
     sequence = Sequence("offer_id_seq")
     return db.session.execute(sequence)
+
+
+def _format_datetime_from_local_timezone_to_utc(date: datetime, local_tz: str) -> datetime:
+    from_zone = tz.gettz(local_tz)
+    to_zone = tz.gettz(utils_date.DEFAULT_STORED_TIMEZONE)
+    datetime_in_tz = date.replace(tzinfo=from_zone)
```

Comment:
(bug) Ça n'est pas la bonne façon de faire : 

```python
>>> dt = datetime.datetime(2022, 6, 24, 12, 0)
>>> print(dt.replace(tzinfo=tz))
2022-06-24 12:00:00+00:09  # oups, l'offset n'est pas bon !
```

La bonne façon de faire est d'utiliser  `localize()` : 


```suggestion
    datetime_in_tz = from_zone.localize(dt)
```

En plus, `localize()` a le bon goût de lever une erreur si l'argument a déjà une timezone.

Reply:
@dbaty j'ai modifié. 
Remarque : la fonction `def utc_datetime_to_department_timezone` utilise le replace (bug potentiel?)

Reply:
Je pense qu'il n'y a pas de bug car il n'y a pas d'offset pour le fuseau horaire UTC. Mais je te laisse vérifier. :)